assets/js/components/ServiceConsent/index.js

```diff
@@ -17,28 +18,51 @@ const getLastUsed = (service, history) => {
 class ServiceConsent extends React.Component {
   constructor(props) {
     super(props)
-    this.deleteConsentClick = this.deleteConsentClick.bind(this)
-  }
+    this.deleteConsent = this.deleteConsent.bind(this)
 
-  deleteConsentClick() {
-    const { service, locale, intl, deleteServiceConsent } = this.props
-    const serviceName = service.name[locale] || service.name['fi']
-    const msg = intl.formatMessage(
-      { id: 'service.consent.delete.confirm' },
-      { service: serviceName }
-    )
-    if (confirm(msg)) {
-      deleteServiceConsent(service.consent)
+    this.state = {
+      show: false
```

Comment:
Could rename to showConfirmation or something a bit more verbose

Reply:
Wonder if I accidentaly left something uncommitted as I do believe I renamed that already